src/components/AddEditMovieForm/AddEditMovieForm.js

```diff
@@ -1,42 +1,73 @@
+import { Form, Formik } from 'formik';
 import PropTypes from 'prop-types';
 import React from 'react';
+import * as Yup from 'yup';
 
+import { availableGenres } from '../../movie-config';
 import { movie } from '../../types/movie';
 import { defaultHandler } from '../../utils/util-func';
+import MultiSelectInput from '../MultiSelectInput/MultiSelectInput';
+import TextInput from '../TextInput/TextInput';
 import './AddEditMovieForm.scss';
 
 const AddEditMovieForm = ({ movie, handleSubmit }) => {
-  const handleSave = (e) => {
-    e.preventDefault();
-    handleSubmit(movie);
-  };
-
   return (
-    <form className="add-edit-movie-form">
-      <div className="title">{movie.id ? 'Edit' : 'Add'} movie</div>
-      {movie.id && (
-        <>
-          <label htmlFor="id">Movie ID:</label>
-          <div>{movie.id}</div>
-        </>
+    <Formik
+      initialValues={{
+        id: movie.id,
+        title: movie.title || '',
+        releaseDate: movie.releaseDate || '',
+        poster: movie.poster || '',
+        genres: movie.genres || [],
+        overview: movie.overview || '',
+        runtime: movie.runtime || '',
+      }}
+      validationSchema={Yup.object({
```

Comment:
it will be better to move validation schema outside this component
but not necessary